spec/controllers/users_controller_spec.rb

```diff
@@ -26,16 +26,16 @@
     end
 
     describe 'Change locale' do
-      it 'updates locale for admin' do
+      before :each do
+        I18n.locale = :en
+      end
+
+      it 'updates locale' do
```

Comment:
This looks like a good candidate for a shared test. Just swap out the `patch_as X` for `patch_as user` and then set the `user` let variable to admin, student, ta.